server/player/player.go

```diff
@@ -623,6 +637,25 @@ func (p *Player) Hurt(dmg float64, src world.DamageSource) (float64, bool) {
 	return totalDamage, true
 }
 
+// applyTotemEffects is an unexported function that is used to handle totem effects.
+func (p *Player) applyTotemEffects() {
+	p.addHealth(1 - p.Health())
```

Comment:
What is the logic here? Is it not supposed to just be `p.addHealth(1)`?

Reply:
technically yes but since I'm calling it before the player gets hurt / killed then I have to do it like this if not they would keep the health they had before they got hurt